platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1712,7 +1712,7 @@ public void newConnectionOpened(final Connection sc) {
      * {@inheritDoc}
      */
     @Override
-    public void connectionClosed(final boolean outbound) {
+    public void connectionClosed(final boolean outbound, Connection conn) {
```

Comment:
I will keep this in mind.   I added the metrics to `NetworkMetrics.java` since there were already connection related metrics in the class and ConnectionTracker was an interface and not a class with internal state.  

What would the aesthetic or principle at play be that would cause me to turn the interface into a class?